modin/pandas/base.py

```diff
@@ -3059,119 +2771,7 @@ def to_hdf(
-        indent=None,
-        storage_options: StorageOptions = None,
-    ):  # pragma: no cover  # noqa: PR01, RT01, D200
-        """
-        Convert the object to a JSON string.
-        """
-        return self._default_to_pandas(
-            "to_json",
-            path_or_buf,
-            orient=orient,
-            date_format=date_format,
-            double_precision=double_precision,
-            force_ascii=force_ascii,
-            date_unit=date_unit,
-            default_handler=default_handler,
-            lines=lines,
-            compression=compression,
-            index=index,
-            indent=indent,
-            storage_options=storage_options,
-        )
-
-    def to_latex(
-        self,
-        buf=None,
-        columns=None,
-        col_space=None,
-        header=True,
-        index=True,
-        na_rep="NaN",
-        formatters=None,
-        float_format=None,
-        sparsify=None,
-        index_names=True,
-        bold_rows=False,
-        column_format=None,
-        longtable=None,
-        escape=None,
-        encoding=None,
-        decimal=".",
-        multicolumn=None,
-        multicolumn_format=None,
-        multirow=None,
-        caption=None,
-        label=None,
-        position=None,
-    ):  # pragma: no cover  # noqa: PR01, RT01, D200
-        """
-        Render object to a LaTeX tabular, longtable, or nested table.
-        """
-        return self._default_to_pandas(
-            "to_latex",
-            buf=buf,
-            columns=columns,
-            col_space=col_space,
-            header=header,
-            index=index,
-            na_rep=na_rep,
-            formatters=formatters,
-            float_format=float_format,
-            sparsify=sparsify,
-            index_names=index_names,
-            bold_rows=bold_rows,
-            column_format=column_format,
-            longtable=longtable,
-            escape=escape,
-            encoding=encoding,
-            decimal=decimal,
-            multicolumn=multicolumn,
-            multicolumn_format=multicolumn_format,
-            multirow=multirow,
-            caption=None,
-            label=None,
-        )
-
-    def to_markdown(
-        self,
-        buf=None,
-        mode: str = "wt",
-        index: bool = True,
-        storage_options: StorageOptions = None,
-        **kwargs,
-    ):  # noqa: PR01, RT01, D200
-        """
-        Print `BasePandasDataset` in Markdown-friendly format.
-        """
-        return self._default_to_pandas(
-            "to_markdown",
-            buf=buf,
-            mode=mode,
-            index=index,
-            storage_options=storage_options,
-            **kwargs,
-        )
-
-    def to_numpy(
-        self, dtype=None, copy=False, na_value=no_default
-    ):  # noqa: PR01, RT01, D200
-        """
-        Convert the `BasePandasDataset` to a NumPy array.
-        """
+    def to_numpy(self, dtype=None, copy=False, na_value=no_default):
```

Comment:
I think because most pandas-inherited methods don't need those `noqa`-s anymore since we introduced `@_inherit_docstrings(pandas.DataFrame, apilink=["pandas.DataFrame", "pandas.Series"])` decoration. I would rather fix all those in a separate issue/PR.

Reply:
ok